client/src/components/CKANRegistryDisplay.js

```diff
@@ -295,20 +300,29 @@ class CKANRegistryDisplay extends Component {
 
     const dataStore = CKANApi.loadDatastore(endpoint, identifier);
 
-    const visibleFields = [
-      ...this.getVisibleFields(),
-    ];
-    if (!visibleFields.find(value => value === '_id')) {
+    const visibleFields = this.getVisibleFields();
+    if (!visibleFields.includes('_id')) {
```

Comment:
It might seem like we can remove this LOC because a side effect of searching distinct per column means that we get this column but that's _only_ when searching distinct by column - it doesn't cover using the basic endpoint or just simply not searching distinct per column.